examples/hosts/app-integration/external-data/src/start.ts

```diff
@@ -35,8 +35,12 @@ const render = (model: IAppModel, showExternalServerView: boolean): void => {
 };
 
 async function start(): Promise<void> {
+	// TODO: Create a form to input the task list id to import from external server.
+	// Hardcoded it here for now.
+	const taskListId = "task-list-1";
```

Comment:
If I got this right and this is the id of the task list in the external service (e.g. JIRA), I might name this `externalTaskListId` for additional clarity that it is not the id of the rendered task list React component.